pkg/scripts/Vagrantfile

```diff
@@ -0,0 +1,55 @@
+Vagrant.require_version ">= 1.8.7"
+unless Vagrant.has_plugin?("vagrant-vbguest")
+  raise "Please install the vagrant-vbguest plugin by running `vagrant plugin install vagrant-vbguest`"
+end
+
+unless Vagrant.has_plugin?("vagrant-cachier")
+  raise "Please install the vagrant-cachier plugin by running `vagrant plugin install vagrant-cachier`"
+end
+
+VAGRANTFILE_API_VERSION = "2"
+
+MEMORY = 4096
+CPU_COUNT = 2
+
+MOUNT_DIRS = {
+  :bigchaindb => {:repo => "bigchaindb", :local => "/opt/stack/bigchaindb", :owner => "edxapp"},
+}
+
+boxname = ENV['BOXNAME']
```

Comment:
you are taking the boxname as a configuration param, but not using it anywhere. It is default to `ubuntu/xenial64`

Reply:
good catch will remove